docs/tutorials/notebooks/api_tiling_tutorial.ipynb

```diff
@@ -234,9 +235,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "solaris",
+   "display_name": "Python 3",
```

Comment:
Once this is ready, let's change the tutorial kernel name to `solaris` for consistency.